libvirt.go

```diff
@@ -225,7 +224,15 @@ func (d *Driver) PreCreateCheck() error {
 }
 
 func (d *Driver) Create() error {
-	if err := mcnutils.CopyFile(d.ImagePath, d.ResolveStorePath(d.MachineName)); err != nil {
+	if d.ImageFormat == "" {
+		d.ImageFormat = "qcow2"
+	}
+	if d.ImageFormat != "qcow2" {
+		return fmt.Errorf("Unsupported VM image format: %s", d.ImageFormat)
+	}
+
```

Comment:
Isn't this contradictory, in the fist statement you check if there is no `imageFormat` and set it to `qcow2` and in the second statement if it is different then `qcow2` then return an unsupported format error, should we just fix it to `qcow2` then for this driver?

Reply:
```
if d.ImageFormat == "" {
	d.ImageFormat = "qcow2"
}
```
I added this for backward compatibility, but I'm not really sure it's needed, I can try to remove it :)
I prefer to error out when `ImageFormat` is not `qcow2` rather than hardcoding it as it's more obvious why setting `d.ImageFormat` to a non-qcow2 image format is not taken into account.